packages/strapi/lib/Strapi.js

```diff
@@ -1,6 +1,14 @@
 'use strict';
 
 // Dependencies.
+const dotenv = require('dotenv');
+
+if(process.env.ENV_PATH) {
```

Comment:
You don't need to do the if statement here :)

Reply:
ok removed it.